KMeans.java

```diff
@@ -0,0 +1,126 @@
+import java.io.*;
+import java.util.*;
+import java.lang.Math; 
+
+public class KMeans {
+
+	private Vector midpoint (List<Vector> listofvectors) { //computes + returns a midpoint Vector from a given list of Vectors
```

Comment:
parameters should be in `lowerCamelCase`. So, `listOfVectors`.  But actually, I'd recommend just `vectors` as the name of this param